mail/mailutils/Portfile

```diff
@@ -28,26 +28,26 @@ checksums           rmd160  0d06563d2af2e1b1c47274cfa5b0127ded180769 \
 # https://trac.macports.org/ticket/62993
 patchfiles          patch-configure-libtool.diff
 
-configure.args      --verbose \
-                    --disable-silent-rules \
```

Comment:
please don't remove `--disabel-silent-rule`, we typically want to use that flag to make it easier to figure out why things potentially don't work

Reply:
thanks for the advice. i removed -disable-silent-rules.
please let me know if I have to remove --verbose as well.

Reply:
i mean i removed the change, leaving --disable-silent-rules in place :-)
--